modules/core/src/v2/coins/cspr.ts

```diff
@@ -0,0 +1,76 @@
+import { BaseCoin, KeyPair, SignedTransaction, VerifyAddressOptions, VerifyTransactionOptions } from '../baseCoin';
+import { NodeCallback } from '../types';
+import { BitGo } from '../../bitgo'; 
+import { Cspr as CsprAccountLib } from '@bitgo/account-lib';
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+
+import Bluebird = require('bluebird');
+
+export class Cspr extends BaseCoin {
+    
+    protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+
+    constructor(bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>) {
+        super(bitgo);
+    
+        if (!staticsCoin) {
+          throw new Error('missing required constructor parameter staticsCoin');
+        }
```

Comment:
Why is the parameter optional if we are going to throw an exception when it's missing?